src/Tree/Strategy/ORM/Nested.php

```diff
@@ -63,22 +71,26 @@ class Nested implements Strategy
      * tree in case few root nodes will be persisted
      * on one flush for node classes
      *
-     * @var array
+     * @var array<string, int>
      */
     private $treeEdges = [];
 
     /**
      * Stores a list of node position strategies
      * for each node by object id
      *
-     * @var array
+     * @var array<int, string>
+     *
+     * @phpstan-var array<int, value-of<self::ALLOWED_NODE_POSITIONS>>|array{}
```

Comment:
do we need the `|array{}` part?

Reply:
hmm I think in that example is necessary because it is described with an array shape, but in this case it is a "general" array, it would be like [this example](https://phpstan.org/r/b852a8b0-2a50-4d01-ba5c-b93528acb17b) I think.

Reply:
You're right. Thank you!